src/mtype.d

```diff
@@ -7073,9 +7113,9 @@ public:
                         VarDeclaration v = s.isVarDeclaration();
                         FuncDeclaration f = s.isFuncDeclaration();
                         if (intypeid || !v && !f)
-                            e = DsymbolExp.resolve(loc, sc, s, false);
+                            e = DsymbolExp.resolve(loc, sc, s, false);      // todo
                         else
-                            e = new VarExp(loc, s.isDeclaration());
+                            e = new VarExp(loc, s.isDeclaration(), false);  // todo
```

Comment:
I've tested a new better fix for `VarExp.hasOverloads` and others' setting. Now it's changed to give `true`, and the commit is properly in #5198.